x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/components/migration_status_panels/migration_panel_title.tsx

```diff
@@ -104,16 +121,48 @@ export const MigrationPanelTitle = React.memo<MigrationPanelTitleProps>(({ migra
                   onClick={togglePopover}
                   aria-label={i18n.OPEN_MIGRATION_OPTIONS_BUTTON}
                   data-test-subj="openMigrationOptionsButton"
-                  isLoading={isUpdatingMigrationName}
+                  isLoading={isUpdatingMigrationName || isDeletingMigration}
                 />
               }
               isOpen={isPopoverOpen}
               closePopover={closePopover}
               panelPaddingSize="none"
               anchorPosition="downCenter"
             >
-              <EuiContextMenuPanel size="s" items={items} />
+              <EuiContextMenuPanel size="s">
```

Comment:
@semd , @kqualters-elastic , should we disable this context menu if user does not have capabilities?. It looks like we are not doing that check anywhere for `rename` and `delete`. Please let me know incase i missed it.

Reply:
And if, for some reason, the user is able to bypass the UI controls, the API won't allow the operation:

![API error](https://github.com/user-attachments/assets/5901eeeb-2727-46b8-a646-f4bc5b4fd868)

But it's a good reminder anyway, we need to keep privileges always in mind. 👍